src/components/banner/MockLocalStorage.tsx

```diff
@@ -0,0 +1,18 @@
+const localStorageMock = (() => {
```

Comment:
This definitely works but it's a little Javascript-y in that we're creating and using arbitrary objects. Can you do it this way: https://stackoverflow.com/a/41434763

Reply:
Fixed